sqlmesh/core/console.py

```diff
@@ -416,26 +416,48 @@ def show_model_difference_summary(
             return
 
         tree = Tree(f"[bold]Summary of differences against `{context_diff.environment}`:")
-        added_model_names = context_diff.added - ignored_snapshot_names
+        added_model_names = {
+            name for name, snapshot in context_diff.added.items() if snapshot.is_model
+        } - ignored_snapshot_names
         if added_model_names:
             added_tree = Tree(f"[bold][added]Added Models:")
             for model_name in added_model_names:
                 added_tree.add(f"[added]{model_name}")
             tree.add(added_tree)
 
-        removed_model_names = context_diff.removed - ignored_snapshot_names
+        added_audit_names = {
```

Comment:
why did you add this to the ui? we don't need to show audits in the console diff

Reply:
Discussed offline